src/services/pact-verifier-service.test.ts

```diff
@@ -37,40 +38,23 @@ describe('PactVerifierService', () => {
     }).compile();
 
     pactVerifierService = moduleRef.get<PactVerifierService>(PactVerifierService);
-    pactVerifierProvider = moduleRef.get(PactModuleProviders.PactVerifier);
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
   });
 
   describe('when configuring the providerBaseHost', () => {
     beforeEach(async () => {
-      options = { ...options, providerHost: '0.0.0.0' };
-      await pactVerifierService.verify(appMock as any);
+      options = { ...options, providerBaseUrl: 'http://127.0.0.1:80' };
+      await pactVerifierService.verify(appMock as unknown as INestApplication);
```

Comment:
`appMock as unknown as INestApplication`. <-- This is risky as it means the test might not be testing what you think it is. It's better than `any`, but it would be ideal to use something that actually satisfies `verify`.

If this is some internal nest thing that's hard to mock, I would personally put this conversion at the definition of `appMock`.

Up to you. I like the current version more than the `any` version.